client/helpers.go

```diff
@@ -302,6 +302,19 @@ func IsAWSError(err error, code ...string) bool {
 	return false
 }
 
+func IgnoreCustomError(err error, code string, messageRegex string) bool {
```

Comment:
Maybe expect `*regexp.Regexp` and use that instead of compiling each time, which is costly. Ideally regexes should be compiled only once per executable invocation (using `regexp.MustCompile()` on a global variable)